manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.5 $debug=QUSU.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Amazon.WorkspacesClient
+PackageVersion: 5.9.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
```

```suggestion
InstallerType: wix
Scope: machine
```
Package installs machine-wide (for all users)

---

Apologies these was just done with quick update with YamlCreate. I assumed it would keep original info

---

Well it should've done so if you used its `Quick Update` functionality. Maybe you didn't update your fork and `git pull` into your local repo as YamlCreate uses the latest 'local' manifest as the base manifest for update. winget-create on the other hand, always fetches the latest manifest from the upstream repo so you won't likely experience that problem there